### ISSUE TITLE
Scaladoc does not show deprecation message

### ISSUE THREAD

**Issue by mwisnicki** [scala/bug#12982]
https://www.scala-lang.org/api/3.4.1/scala/reflect/ClassManifestDeprecatedApis.html shows:

> Deprecated true

https://github.com/scala/scala/blob/80514f73a6c7db32df9887d9a5ca9ae921e25118/src/library/scala/reflect/ClassManifestDeprecatedApis.scala#L21

```scala
@deprecated("use scala.reflect.ClassTag instead", "2.10.0")
```

**Comment by som-snytt** [comment]
Please open a ticket at https://github.com/scala/scala3/issues

By coincidence, I recently looked at Scaladoc and missed the visible markers of deprecation from Scaladoc 2.

**Comment by mwisnicki** [comment]
Done, https://github.com/scala/scala3/issues/20118

BTW bug template of this repository references wrong repo for Scala 3

**Comment by som-snytt** [comment]
@SethTisue ☝️ the scala3 link is stale in the issue template

**Comment by SethTisue** [comment]
> bug template of this repository references wrong repo for Scala 3

thanks for noticing; fixed: 4b731289e24d77b5e2594b05177d3d9c36f18bce
